src/ripple/rpc/handlers/GetCounts.cpp

```diff
@@ -90,11 +90,31 @@ Json::Value doGetCounts (RPC::Context& context)
     textTime (uptime, s, "second", 1);
     ret["uptime"] = uptime;
 
-    ret["node_writes"] = app.getNodeStore().getStoreCount();
-    ret["node_reads_total"] = app.getNodeStore().getFetchTotalCount();
-    ret["node_reads_hit"] = app.getNodeStore().getFetchHitCount();
-    ret["node_written_bytes"] = app.getNodeStore().getStoreSize();
-    ret["node_read_bytes"] = app.getNodeStore().getFetchSize();
+    auto counters = app.getNodeStore().counters();
+    std::uint32_t stores[2];
+    *stores = counters.stores.load();
+    ret["node_writes_h"] = stores[0];
+    ret["node_writes_l"] = stores[1];
```

Comment:
This is absolutely the kind of interface I had in mind for returning a 64-bit value.  Do you think it will meet your needs?  That's the most important part.